gramps/gui/configure.py

```diff
@@ -1496,13 +1496,13 @@ def database_backend_changed(self, obj):
 
     def set_connection_widgets(self, db_choice):
         """
-        Sets the connection widgets sensitive for PostgreSQL.
+        Sets the connection widgets insensitive for embedded databases.
         """
         for widget in self.connection_widgets:
-            if db_choice == 'postgresql':
-                widget.set_sensitive(True)
-            else:
+            if db_choice in ('bsddb', 'sqlite'):
                 widget.set_sensitive(False)
```

Comment:
I have to wonder if the unsophisticated user really needs to see these fields at all when running on the standard db.  I would think it would only cause confusion when there are fields present that never seem to do anything.  Perhaps make them not visible instead?

Reply:
This PR doesn't make any changes to the user interface.